osfoffline/polling_osf_manager/osf_query.py

```diff
@@ -17,10 +17,11 @@
 
 
 class OSFQuery(object):
-    def __init__(self, loop, oauth_token):
+    def __init__(self, loop, oauth_token, limit=5):
```

Comment:
:smile_cat: 

Since we presently use 8 workers in production AFAIK, perhaps a more reasonable limit would be 4? It's a small thing, but two people syncing at the same time could saturate the server wherein one person would be getting better service (more requests) than the other at a given time.